Back-end/prisma/schema.prisma

```diff
@@ -11,15 +11,17 @@ model User {
   id          Int        @id @default(autoincrement())
   username    String     @unique
   password    String
-  Headline    String
-  FirstName   String
-  LastName    String
   accountType String
   date        DateTime   @default(now())
+  FirstName   String
+  LastName    String
+  Headline    String
+  state       String
+  age         Int
```

Comment:
Unify whether capitalize the schema name or not.

Reply:
Spoke to Alisa that if I try to unify the schema names, it will force me to delete data from my database.